test/libponyc-run/bare-function-use-of-void-pointers/main.pony

```diff
@@ -0,0 +1,11 @@
+use @printf[I32](fmt: Pointer[None] tag, ...)
```

Comment:
This can be removed

Reply:
yes, forgot to delete from previous test version